packages/tokens/non-fungible/src/extensions/consecutive/storage.rs

```diff
@@ -41,11 +43,22 @@ impl ContractOverrides for Consecutive {
     }
 }
 
-/// Storage keys for the data associated with `FungibleToken`
+/// Maximum number of vectors holding bitfields, which denote ids
```

Comment:
I believe a visual demonstration via comments would be super helpful here. Something like this:

```
Bucket 2
├── Item 0 → bits for Token IDs 512..575
├── Item 1 → bits for Token IDs 576..639
├── Item 2 → bits for Token IDs 640..703
├── Item 3 → bits for Token IDs 704..767
```

If this is erroneous, feel free to fix it, or apply any other format you like :) 
This was just an example

Reply:
like that 👌